packages/cli/tests/watch.test.js

```diff
@@ -114,23 +265,22 @@ describe('should determine the correct port', () => {
 
 	it('should fallback to random if $PORT or 8080 are taken and --port is not specified', async () => {
 		process.env.PORT = '4004';
-		await Promise.all([determinePort(), determinePort()]).then(values => {
-			expect(values[0]).toBe(4004);
-			expect(values[1]).toBeGreaterThanOrEqual(1024);
-			expect(values[1]).toBeLessThanOrEqual(65535);
-		});
+		const ports = await Promise.all([determinePort(), determinePort()]);
+		expect(ports[0]).toBe(4004);
+		expect(ports[1]).not.toBe(4004);
+		expect(ports[1]).toBeGreaterThanOrEqual(1024);
+		expect(ports[1]).toBeLessThanOrEqual(65535);
 
 		// This is pretty awful, but would be the way to do it. get-port locks the port for ~30 seconds,
```

Comment:
do we want to keep this commented out stuff?

Reply:
No, it should go. I believe I was wanting to salvage that but never could figure out how to do it without waiting obscene amounts of time. Thanks